frontend/components/molecules/Nav.js

```diff
@@ -51,39 +58,43 @@ const Nav = ({ navItems, index }) => {
   });
 
   return (
-    <MyNav
-      display="flex"
-      flexDirection="column"
-      alignItems="flex-end"
-      bg="greys.0"
-      shadow={scrolled ? 'default' : 'none'}
-    >
-      <SystemComponent display="flex" justifyContent="flex-end">
-        <NavLogo alignSelf="center" />
-      </SystemComponent>
-      <SystemComponent
+    <NavOutterLayer>
+      <MyNav
         display="flex"
-        flexWrap="wrap"
-        justifyContent="flex-end"
-        mt={3}
-        width={['80%', '80%', '80%', 'auto']}
+        flexDirection="column"
+        alignItems="flex-end"
+        bg="greys.0"
+        shadow={scrolled ? 'default' : 'none'}
       >
-        {navItems.map(({ name, link }, i) => (
-          <a href={link} key={i}>
-            <NavLink
-              fontSize={['smallNav', 'smallNav', 'smallNav', 'nav']}
-              fontWeight="bold"
-              ml={7}
-              color={i === index ? 'theme' : 'foreground'}
-              display={['none', 'block']}
-            >
-              {name}
-            </NavLink>
-          </a>
-        ))}
-        {router.pathname.startsWith('/login') ? null : <LogoutButton />}
-      </SystemComponent>
-    </MyNav>
+        <SystemComponent display="flex" justifyContent="flex-end">
+          <NavLogo alignSelf="center" />
+        </SystemComponent>
+        <SystemComponent
+          display="flex"
+          flexWrap="wrap"
+          justifyContent="flex-end"
+          mt={3}
+          width={['80%', '80%', '80%', 'auto']}
+        >
+          {navItems.map(({ name, link }, i) => (
+            <a href={link} key={i}>
```

Comment:
Could you confirm that there's no warning in the browser console that says 
`Warning: Each child in an array should have a unique "key" prop.`

related [read](https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js/43892905#43892905)

Reply:
I think Gordon fixed this now. He tested this locally to make sure there is no warning anymore. Thank you.